tests/scripts/thread-cert/node.py

```diff
@@ -1936,6 +1937,7 @@ def router_table(self):
                 'lqout': lqout,
                 'age': age,
                 'emac': emac,
+                'link': link,
```

Comment:
```suggestion
                'linked': linked,
```

Reply:
I will keep it as is to be inline with `router <id>` command.